nilearn/plotting/displays.py

```diff
@@ -1042,6 +1056,13 @@ def add_graph(self, adjacency_matrix, node_coords,
                 size(s) of the nodes in points^2.
             edge_cmap: colormap
                 colormap used for representing the strength of the edges.
+            edge_vmin: float, optional, default: None
+            edge_vmax: float, optional, default: None
+                If not None, either or both of these values will be used to
```

Comment:
might not be explicit enough to convey the effect of `edge_vmin/max`on the plot...